controllers/account.controller.js

```diff
@@ -0,0 +1,29 @@
+const getAllAccounts = (req, res) => {
+    res.json({message: "GET account"});
+}
+
+
+const addAccount = (req, res) => {
+    res.json({message: "POST account"})
+}
+
+const deleteAccountById = (req, res) => {
+    const id = req.params.id;
+   
+    res.json({message: "DELETE account"});
+}
+
+const updateAccountById = (req, res) => {
+    const id = req.params.id;
+
```

Comment:
Try to keep the same formatting everywhere. empty lines here looks not very good.
In one of the next lectures you will know hot to format code automatically.